src/layouts/Layout.astro

```diff
@@ -1,10 +1,17 @@
 ---
+import "normalize.css";
+import "magick.css";
+
 interface Props {
   title: string;
+  fiscalSponsor: string;
 }
 
 const { title } = Astro.props;
-import "../styles/globals.css";
+
+const fiscalSponsor = Astro.props.fiscalSponsor === "true" ? true : false;
```

Comment:
Can't we just have it just static on just /index? so we don't have to add that prop to every page instead of using `fiscalSponsor`

Reply:
i can make it an optional prop so its not required; would that work?